src/components/Sidebar/contentTree.js

```diff
@@ -78,6 +84,29 @@ const style = theme => css`
 
 const extractLevelGroupID = pathname => pathname.replace(/(^\/|\/$)/g, '').split('/');
 
+const getDeepNavigation = (navItemGroup, navLinkGroup, pathname, deepLevel) => {
+    if(deepLevel > limitDeepLevel) 
+      return [];
+
+    // Exit recursive when children.length = 0
+    navItemGroup.children.map( childNavItemGroup => {
+     const isAccessByUrl = childNavItemGroup.url.indexOf(extractLevelGroupID(pathname)[deepLevel]) > 0;
+      const navLinkGroupTmp = {
+        name: childNavItemGroup.title,
+        url: childNavItemGroup.url,
+        key: childNavItemGroup.url,
+        isExpanded: isAccessByUrl
+      }
+
+      const deepNavigationLinks = getDeepNavigation(childNavItemGroup, [], pathname, deepLevel + 1);
+      if(deepNavigationLinks.length > 0) 
```

Comment:
Very minor I think if without curry braces is okay for early exit but for doing something else it can cause a bug later if someone forget about this.

Please update this if block to :
```js

      if(deepNavigationLinks.length > 0) {
         navLinkGroupTmp.links = deepNavigationLinks;
      }
```